src/runningMethods/recovery.js

```diff
@@ -13,11 +13,12 @@ module.exports = async () => {
         fn.dataSources.aka,
         fn.dataSources.es,
         fn.dataSources.ads, 
-        fn.dataSources.adNN, 
+        /*fn.dataSources.adNN, 
```

Comment:
remove commeneted dataSOurces

Reply:
remove or uncomment?

Reply:
we want them of course, keep them just remove the comments